CHANGELOG.md

```diff
@@ -5,10 +5,11 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 
--   #1689 : Add Python support for list method `append()`.
--   #1692 : Add Python support for list method `insert()`.
--   #1690 : Add Python support for list method `pop()`.
--   #1691 : Add Python support for list method `clear()`.
+-   #1689 : Add Python support for list method `append()`
+-   #1692 : Add Python support for list method `insert()`
+-   #1694 : Add Python support for list method `extend()`
+-   #1690 : Add Python support for list method `pop()`
+-   #1691 : Add Python support for list method `clear()`
```

Comment:
In English sentences should end with a `.`. Is there any reason this has been removed from all these lines?